LEAF_Nexus/api/controllers/NationalEmployeeController.php

```diff
@@ -66,10 +66,9 @@ public function post($act)
 
                 require_once __DIR__ . "/../../sources/Employee.php";
                 require_once __DIR__ . "/../../sources/Login.php";
-                require_once __DIR__ . "/../../config.php";
                 require_once __DIR__ . "/../../db_mysql.php";
 
-                $config = new Orgchart\Config();
+                global $config;
```

Comment:
The global variable is a synonym for bad practice. Refrain from using it.

Reply:
The problem here is a question of time. We can go through the process of passing the config variable into every constructor and making sure that every call to the classes in question passes it appropriately, but this would take a lot longer to code and test. Or we can declare that we're using the shared global config that the rest of the application is using and trust that our devs wont try to overwrite a variable called $config.